great_ape_safe/ape_api/balancer.py

```diff
@@ -280,6 +280,20 @@ def stake_all(self, pool, destination=None, dusty=False):
         self.stake(pool, mantissa, destination, dusty)
 
 
+    def unstake(self, pool, mantissa, claim=True):
+        gauge = self.safe.contract(self.gauge_factory.getPoolGauge(pool))
+        bal_pool_before = pool.balanceOf(self.safe)
+        gauge.withdraw(mantissa, claim)
+        assert pool.balanceOf(self.safe) >= bal_pool_before
```

Comment:
See comment on Unstake all, if 1-1 then we can have a stricter check

Reply:
sorted on #606 as well